people.json

```
@@ -86,6 +86,51 @@
          ]
       ]
    },
+   
+   {
+      "name": "Dan-Dylan Lavictoire",
+      "image": "https://cdna.artstation.com/p/users/avatars/000/678/892/large/9850decab6e8f4a20895d228cd97e156.jpg?1563831406",
+      "skills": [
+         "game design",
+         "programming",
+         "production",
+         "narrative design",
+         "game audio",
+         "art",
+      ],
+      "location": "France",
+      "websites": {
+         "personal": [
+            [
+               "Website",
+               "https://danlavictoire.wixsite.com/dadydoux/about"
+            ],
+            [
+               "Twitter",
+               "https://twitter.com/RoiDaDyDoux"
+            ],
+           
```

another sneaky comma
```suggestion
            ]
```